src/charm.py

```diff
@@ -147,7 +170,7 @@ def _command():
                         "override": "replace",
                         "summary": "avalanche service",
                         "startup": "enabled",
-                        "command": _command(),
+                        "command": _command().rstrip(),
```

Comment:
Why are we returning a string that needs stripping immediately after building it?